### ISSUE TITLE
3.3.7: Clarify exceptions in definition

### ISSUE THREAD

**Issue by alastc** [w3c/wcag#1310]
The definition of Cognitive function test is:
> task that requires the user to remember, manipulate, or transcribe information.
> Examples include, but are not limited to:
> - memorization, such as remembering a username, password, set of characters, images, or patterns;
> - transcription, such as typing in characters;
> - use of correct spelling;
> - performance of calculations;
> - solving of puzzles.
> Note: Remembering your own name, email address, or phone number is not considered a cognitive function test.

Comment from Gregg Vanderheiden about the name & email address aspect:
> If those are exceptions - they need to be added to the definition proper of cognitive test.

The rational was that for _your_ information, you only have to remember one name/email/phone number. Things like usernames and passwords are (or at least should be!) per-site, so you would have to remember many such things.

I can see two solutions to this, either make the 1st part more specific, e.g:
> task that requires the user to remember, manipulate, or transcribe information for the website.

Or add it to the 1st bullet, e.g:
> > - memorization, such as remembering a username, password, set of characters, images, or patterns. Information not specific to the website (such as your name, email and phone number) are not considered memorization.
